src/ecalc/libraries/libecalc/common/tests/input/test_file_io.py

```diff
@@ -100,6 +100,17 @@ def facility_resource_missing_value_file(tmp_path):
     return facility_file
 
 
+def create_csv_from_line(tmp_path: Path, csv_line: str) -> Path:
+    csv_file = tmp_path / "csv_file.csv"
+    csv_file.write_text(csv_line)
+    return csv_file
+
+
+@pytest.fixture
+def get_tmp_path(tmp_path: Path) -> Path:
```

Comment:
I like that the fixtures just wraps a regular function though, making it potentially possible to reuse the function in more places than just the fixture :)

Reply:
btw, is this actually needed? Is tmp_path a pytest magic fixture? Just use it straight away instead or "rewrapping" it in a fixture?

Reply:
tmp_path is a pytest thing, I had to wrap it to make it work